aea/cli/test.py

```diff
@@ -491,63 +549,20 @@ def find_component_directory_from_component_id_in_registry(
     raise ValueError("Package {} not found.".format(component_id))
 
 
-def aggregate_coverage(coverage_data: List[str], covrc_file: Path) -> None:
-    """Aggregate coverage reports."""
-
-    click.echo("Generating coverage reports.")
-
-    coverage_data = [file for file in coverage_data if Path(file).exists()]
-    coverage(argv=["combine", f"--rcfile={covrc_file}", *coverage_data])
-    coverage(argv=["html", f"--rcfile={covrc_file}"])
-    coverage(argv=["xml", f"--rcfile={covrc_file}"])
-
-    # remove redundant coverage data
-    for file in coverage_data:
-        if Path(file).exists():
-            os.remove(file)
```

Comment:
No need for this function since we will append the coverage with the previous packages when running the `aea test packages` command rather than starting clean